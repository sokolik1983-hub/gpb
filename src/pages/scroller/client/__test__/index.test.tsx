import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { ClientScrollerPage } from '..';

describe('ClientScrollerPage', () => {
  it('renders', () => {
    const wrapper = shallow(<ClientScrollerPage />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
