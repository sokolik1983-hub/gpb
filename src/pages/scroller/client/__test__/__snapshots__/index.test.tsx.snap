// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClientScrollerPage renders 1`] = `
<t>
  Клиентский скроллер
</t>
`;
